media-video/shotcut/shotcut-20.07.11.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils xdg
+
+DESCRIPTION="A free, open source, cross-platform video editor"
+HOMEPAGE="https://www.shotcut.org/ https://github.com/mltframework/shotcut/"
+SRC_URI="https://github.com/mltframework/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+"
+COMMON_DEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5[widgets]
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsql:5
+	dev-qt/qtwebkit:5
```

Comment:
I can't compile this but I trust this ebuild works. (apparently gcc-10 issue, not gonna try again).

But can you poke upstream to deprecate qtwebkit since it's dead and move on to qtwebengine?

Reply:
Probably because of it, I have ~amd64 and gcc-10.2, just did a rebuild without ccache. Not worthing bumping, already reported the situation here: [https://bugs.gentoo.org/688850#c1](https://bugs.gentoo.org/688850#c1) I will have a look at the suggested resolution with the 9999 package.